WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticsEventHandler.cs

```diff
@@ -270,6 +271,10 @@ private static void SetHttpProperties(Activity activity, DependencyTelemetry dep
                 {
                     status = tag.Value;
                 }
+                else if (tag.Key == "otel.status_code")
```

Comment:
If customer use older version of Azure SDK with latest application insights SDK (with this change). Does it break backward compatibility as `otel.status_code` does not exist with older version of Azure SDK?

Reply:
Yes, I'll think about how can we make this backward compatible.

Reply:
Done, we only ignore the default calculation if otel.status_code is available.